pkg/reader/ccip.go

```diff
@@ -667,6 +669,59 @@ func (r *ccipChainReader) GetRMNRemoteConfig(
 	}, nil
 }
 
+// GetRmnCurseInfo returns rmn curse/pausing information about the provided chains
+// from the destination chain RMN remote contract.
+func (r *ccipChainReader) GetRmnCurseInfo(
+	ctx context.Context,
+	sourceChainSelectors []cciptypes.ChainSelector,
+) (*rmntypes.CurseInfo, error) {
+	if err := validateExtendedReaderExistence(r.contractReaders, r.destChain); err != nil {
+		return nil, fmt.Errorf("validate dest=%d extended reader existence: %w", r.destChain, err)
+	}
+
+	type retTyp struct {
+		CursedSubjects [][16]byte
+	}
+	var cursedSubjects retTyp
+
+	err := r.contractReaders[r.destChain].ExtendedGetLatestValue(
```

Comment:
I assume you tested this; the named returns have messed up contract reader queries before